server-spi-private/src/test/java/org/keycloak/broker/provider/util/SimpleHttpTest.java

```diff
@@ -64,7 +65,7 @@ public static Collection<Object[]> entities() {
         @Test
         public void withCharset() throws IOException {
             HttpResponse httpResponse = createBasicResponse(entity);
-            SimpleHttp.Response response = new SimpleHttp.Response(httpResponse);
+            SimpleHttp.Response response = new SimpleHttp.Response(httpResponse, HttpClientProvider.DEFAULT_MAX_CONSUMED_RESPONSE_SIZE);
```

Comment:
I think we could add a test here too. I did something quick in this commit: https://github.com/rmartinc/keycloak/commit/1e6562339d8296c77f6a84fba2181be9f198c2ff
You can use it or something similar.